index.html

```diff
@@ -8,22 +8,239 @@
     <link rel="stylesheet" href="./scss/index.css">
 </head>
 <body>
```

Comment:
-  Kindly above the navigation there are social media links witch have you missed it, Please add it to have social contact and match the given design


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/217829991-5998c092-0c31-4fa5-a417-106e7ebb701b.png" width="600">|<img src="https://user-images.githubusercontent.com/52010789/217830233-9cddf01f-8b7e-4b2d-8838-c2bbb117f13d.png" width="600">|


😎️

🤝️